surrealdb/exceptions/__init__.py

```diff
@@ -0,0 +1,18 @@
+"""
+Copyright © SurrealDB Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from .surreal_exception import SurrealException
+
+__all__ = ("SurrealException",)
```

Comment:
why not just have a file for exceptions? I think a folder is overkill.

Reply:
true, fixed